boards/arm64/fvp_baser_aemv8r/fvp_baser_aemv8r.yaml

```diff
@@ -5,10 +5,9 @@ identifier: fvp_baser_aemv8r
 name: FVP Emulation FVP_BaseR_AEMv8R
 arch: arm64
 type: sim
+simulation: arm_fvp
 toolchain:
   - zephyr
   - cross-compile
 ram: 2048
 flash: 64
-testing:
```

Comment:
Does the v8r have any different core arch paths compared to v8?

It might make sense to have this as `testing/default:true`

Reply:
The major difference is that v8R uses MPU, v8A MMU. And at this point I'm a bit lost of the meaning of `testing/default'.

Reply:
Probably good for v8r to have `testing/default: true` as its enough of a different variant.